.github/dependabot.yml

```diff
@@ -2,10 +2,19 @@ version: 2
 updates:
   - package-ecosystem: "npm"
     directory: "/"
+    commit-message:
+      prefix: "dependabot"
```

Comment:
label?

Reply:
Something like:

```yaml
labels:
  - npm
  - dependencies
```

While keeping commit-message? Not that `dependencies` is the default applied label, could switch to `dependabot` if we want different labels for scala-steward (which doesn't use labels yet)

Reply:
NPM:
```
labels:
  - npm
  - dependencies
```
And keep the default prefixes, it seems to use `chore(deps)`/`chore(deps-dev)` by default - see e.g. #10.

GH actions:
```
labels:
  - github-actions
  - dependenci
commit-message:
  prefix: "chore"
```

What do you think?

Reply:
`dependencies` I guess :stuck_out_tongue: 

Seems fine

Reply:
the label dependencies is the default anyway if you don't specify anything

Reply:
i like the chore prefix too